flash/core/finetuning.py

```diff
@@ -215,3 +215,56 @@ def __init__(
         train_bn: bool = True,
     ):
         super().__init__(FinetuningStrategies.UNFREEZE_MILESTONES, strategy_metadata, train_bn)
+
+
+class FlashDeepSpeedFinetuning(FlashBaseFinetuning):
+    """FlashDeepSpeedFinetuning can be used to create a custom Flash Finetuning Callback which works with
+    DeepSpeed.
+
+    DeepSpeed cannot store and load its parameters when working with pytorch-lightning. So FlashDeepSpeedFinetuning
+    override `_store` not to store its parameters.
```

Comment:
```suggestion
    DeepSpeed cannot store and load its parameters when working with Lightning. So FlashDeepSpeedFinetuning
    overrides `_store` to not store its parameters.
```

We now have Lightning :)